integration-tests/WRITING_TESTS.md

```diff
@@ -0,0 +1,79 @@
+# Writing integration tests for snappy
+
+The integration tests for snappy are written using
+[gocheck](https://labix.org/gocheck) and executed with `adt-run` through `ssh`.
```

Comment:
I'm not sure what you are after.
I added a little line mentioning that integration-tests/main.go passes the ip to adt-run, which executes the tests through ssh. I don't want to document a lot about how adt-run works, because it has nice docs.
If you follow the README, you would be reproducing what we do to run the tests in CI, and to launch the tests manually.